.travis.yml

```diff
@@ -1,12 +1,27 @@
-language: nodejs
-node_js:
-  - 8
+language: python
+cache: pip
+python:
+  - 2.7
 install:
-  - npm install -g markdownlint-cli --save-dev
-before_script:
-  - markdownlint .
+  - pip install Pygments setuptools docutils mock pillow alabaster commonmark recommonmark mkdocs
 script:
-  - true  # add other tests here
+# to be uncommented when the initial content committed.
+#  - mkdocs build --clean --site-dir _build/html --theme readthedocs 
+  - true
+
+matrix:
+  include:
+  ## linters for json and JavaScript files
+    - language: node_js
```

Comment:
what's the plan exactly with nodejs and what with python?

Reply:
python is to test the build using same code as readthedocs
nodejs for the linters